docs/Troubleshooting.md

```diff
@@ -37,8 +37,8 @@ localhost                  : ok=52   changed=12   unreachable=0    failed=1    s
   ```
 
 ### Diagnosis:
-A release of sassoftware/viya4-monitoring-kubernetes prior to 1.2.0 was run by a release of sassoftware/viya4-deployment at release 4.13.0 or later.
-Releases of sassoftware/viya4-monitoring-kubernetes prior to 1.2.0 do not support the installation of OpenSearch logging software which sassoftware/viya4-deployment 4.13.0 or later will attempt to install.
+The cluster-logging task of sassoftware/viya4-monitoring tried to execute code from a release of sassoftware/viya4-monitoring-kubernetes prior to 1.2.0 using a release of sassoftware/viya4-deployment at release 5.0.0 or later.
```

Comment:
I think this sentence should be re-worked a bit to improve clarity.  Maybe update the sentence to something like: 
_The cluster-logging task tried to deploy an older, incompatible, release of sassoftware/viya4-monitoring-kubernetes (i.e. a release earlier than version1.2.0) using a release of sassoftware/viya4-deployment at release 5.0.0 or later._